internal/k8s/user_origin_test.go

```diff
@@ -129,6 +133,15 @@ func (s *userOriginSuite) Test_cdnIngressesForUserOrigins_InvalidAnnotationValue
 			name:            "Invalid YAML",
 			annotationValue: "*",
 		},
+		{
```

Comment:
I added that test, just not explicitly 😃. If you check some of the tests are not passing the new field, but it's in the expected values with the default value.

Reply:
Ah gotcha. LGTM then, but nit: adding a specific test case could make tests more maintainable. You call though

Being tested implicitly will probably remain ok indefinitely, but if we ever remove the test that does it, we won't think to add another to test just the defaults because it probably won't be clear that this is also being tested there.